packages/core/src/App/Containers/Layout/header/dashboard-header-dropdown.jsx

```diff
@@ -0,0 +1,125 @@
+import PartnershipProgrammes from 'Assets/SvgComponents/header/partnership-programmes.svg';
+import PaymentMethods from 'Assets/SvgComponents/header/payment-methods.svg';
+import RegulatoryInfo from 'Assets/SvgComponents/header/regulatory-info.svg';
+import WhyChooseUs from 'Assets/SvgComponents/header/why-choose-us.svg';
+
+const getNavigationItems = type => {
+    const deriv_url = 'https://deriv.com';
+    if (type === 'about') {
+        return [
+            {
+                name: 'Our story',
+                to: `${deriv_url}/about/#story`,
+                icon: <OurStory />,
+            },
+            {
+                name: 'Our leadership',
+                to: `${deriv_url}/about/#leadership`,
+                icon: <OurLeadership />,
+            },
+            {
+                name: 'Regulatory information',
+                to: `${deriv_url}/regulatory/`,
+                icon: <RegulatoryInfo />,
+            },
+            {
+                name: 'Why choose us?',
+                to: `${deriv_url}/why-choose-us/`,
+                icon: <WhyChooseUs />,
+            },
+            {
+                name: 'Partnership programmes',
+                to: `${deriv_url}/partners/`,
+                icon: <PartnershipProgrammes />,
+            },
+            {
+                name: 'Contact us',
+                to: `${deriv_url}/contact_us/`,
+                icon: <ContactUs />,
+            },
+            {
+                name: 'Careers',
+                to: `${deriv_url}/careers/`,
+                icon: <Career />,
+            },
+        ];
+    }
+    if (type === 'resources') {
+        return [
+            {
+                name: 'Help centre',
+                to: `${deriv_url}/help-centre/`,
+                icon: <HelpCentre />,
+            },
+            {
+                name: 'Community',
+                to: `${deriv_url}/help-centre/`,
+                icon: <Community />,
+            },
+            {
+                name: 'Payment methods',
+                to: `${deriv_url}/payment-methods/`,
+                icon: <PaymentMethods />,
+            },
+            {
+                name: 'Blog',
+                to: `${deriv_url}/help-centre/`,
+                icon: <Blog />,
+            },
+        ];
+    }
+    return [];
+};
+
+const HeaderDropdownItem = ({ icon, name, to }) => {
+    return (
+        <a href={to} className='dashboard-header-dropdown__card-link'>
+            <div className='dashboard-header-dropdown__card-link__icon'>{icon}</div>
+            <Text size='s'>{name}</Text>
+        </a>
+    );
+};
+
+const HeaderDropdown = ({ current_ref, onClickHandler, parent, setRef }) => {
+    const [left_offset, setLeftOffset] = React.useState(current_ref?.offsetLeft);
+    const dropdown_container_ref = React.useRef(null);
+
+    const updateOffsets = () => {
+        if (current_ref) {
+            setLeftOffset(current_ref.offsetLeft);
+        }
+    };
```

Comment:
You should wrap this in a `React.useCallback` as the function is re-created on each render and it may be invalid once the cleanup effect runs (on lines 107-109). This will ensure that the reference to the `updateOffsets` callback is the same for each render and will allow us to correctly remove it later on (on line 108).